src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -1607,7 +1608,7 @@ public void keyReleased(final KeyEvent e) {}
     };
   }
 
-  private int computeScrollSpeed(final KeyEvent e) {
+  public int computeScrollSpeed(final InputEvent e) {
```

Comment:
We have a code cohesion problem where this method is used in pretty arbitrary locations. Just updating this method to have higher vis I don't think is the right fix. Can you take a look to see if we can conslidate this functionality entirely to Mappanel, or alternatively conslidate it in TripleAFrame (which is not the right place for it, but at least the code is together in that case)